accountmgmt/go_mod_tidy_hack.go

```diff
@@ -0,0 +1,4 @@
+package accountmgmt
```

Comment:
I am not 100% sure, but I think this file was only needed for packages that pre-existed the multi-module system in place. We can leave it in as it won't make a difference, but I do not think that it is needed.

I think you should be able to confirm this by:

1. Remove this file
2. Push to some branch without the file.
3. Run `go get`:

```shell
$ go get github.com/redhat-developer/app-services-sdk-go/accountmgmt@ams-sdk
go: downloading github.com/redhat-developer/app-services-sdk-go/accountmgmt v0.0.0-20220105151805-20fc728e2f7d
go get: added github.com/redhat-developer/app-services-sdk-go/accountmgmt v0.0.0-20220105151805-20fc728e2f7d
```

I also think we can remove it from the other modules also, it was just needed for the first version for each.

Reply:
Good news. Thank you

Reply:
Will also clean up others after testing it